docs/_articles/en/apex/apex-tasks.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Apex Tasks
+lang: en
+---
+
+This repository contains sample Apex tasks that you can complete in Code Builder or VS Code Extensions for desktop. We constantly add to this list, so make sure you visit this topic often. 
+
+
+## Use Anonymous Apex to Mass Update Account Ratings
+
+Say you have accounts with `Rating` set to `Cold` because of unfavorable business conditions. You now want to make mass updates to these accounts and change their rating to `Warm`.  Say you want to limit this update to mid-sized accounts with more than a certain amount of revenue. 
```

Comment:
```suggestion
Say you have accounts with `Rating` set to `Cold` because of unfavorable business conditions. You now want to make mass updates to these accounts and change their rating to `Warm`.  You want to limit this update to mid-sized accounts with more than a certain amount of revenue. 
```

Reply:
I think two sentences staring with "say" is overkill -- the first one establishes that this is the hypothetical use case.